Sources/NetworkService/InternalError/InternalError.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+/// Defines a set of internal errors that can occur during using the `NetworkService`.
+public enum InternalError: Error {
```

Comment:
```suggestion
public enum NetworkServiceInternalError: Error {
```
I think we should be more specific.

Reply:
Fixed with https://github.com/ios-course/simple-network-service/pull/13/commits/25f1dc555aea1d6a52572303e1f005d090835595